modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/processor/OperationMessageProcessor.java

```diff
@@ -162,36 +111,13 @@ private OperationContextAdapter createOperationContext(ConfigurationInstance<Obj
     @Override
     public void initialise() throws InitialisationException
     {
+        validateComponentConfiguration();
```

Comment:
this code is not equivalent to the one being replaced. It is possible for the correct configuration provider to not be availabel at initialisation time which is why the getConfiguration() method you're removing has an orElseGet() call. Revert the changes associated to that piece. Leave each component type to resolve that on its own way